src/color.cpp

```diff
@@ -530,7 +530,7 @@ nc_color color_from_string(const std::string &color)
         new_color = "c_" + new_color;
     }
 
-    const std::pair<std::string, std::string> pSearch[2] = {{"light_", "lt"}, {"dark_", "dk"}};
+    const std::pair<std::string, std::string> pSearch[2] = {{"light_"}, {"dark_"}};
```

Comment:
This needs to be removed, it converted light_ and dark_ back to lt and dk.

Reply:
Yes, I missed that.

Reply:
Or swap them so it converts the other way round for any stray lt and dk that come in from external mods for compatibility.